### PR TITLE
Feature/update dependencies

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,13 +28,14 @@
...
+    "hardhat": "^2.22.6",
     "prettier": "^3.0.3",
     "prettier-plugin-solidity": "^1.1.3",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.9.3"
+    "@openzeppelin/contracts": "^4.9.3",
+    "dotenv": "^16.4.5"
```
Comment (PierreJeanjacquot) [inline package.json]: if `hardhat` is a dev dependency, `dotenv` should be a dev dependency as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Next
+
+### Changed
+
+- upgrade dependencies
+
```
Comment (PierreJeanjacquot) [inline CHANGELOG.md]:
```suggestion
## Next

### Added

- load environment variables from `.env` file

### Changed

- upgrade dependencies

```

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
see comments and suggestions

File: package.json (hunk 1 above)
Comment (Le-Caignec) [reply package.json]: Fix : Commit: ec39941fb2eb82da57e68c634bd2beffc0b79631

**Review by PierreJeanjacquot — APPROVED** [review]

**Pull request merged**